### PR TITLE
Catch config_block exception and print it

### PR DESCRIPTION
**Pull request by laurennlam**
Catch config_block exception  from the main conf file when an included config file path is not found

### CODE REVIEW

**Comment by Purg** [comment]
I've had no part in the GUI stiff, but looks innocuous and fine to me. Maybe prefix the warning by mentioning the warning/error has to do with the configuration?

**Comment by mleotta** [comment]
I agree with @Purg, it looks good to me, unless we want to take it further and make the message more verbose or possibly even add the dialog.  @mwoehlke-kitware, any thoughts before I merge this?

**Comment by mwoehlke-kitware** [comment]
The intent, from what I can tell, is that `Project` should have an API to retrieve the error, when one occurs (see e.g. [QIODevice::errorString()](http://doc.qt.io/qt-5/qiodevice.html#errorString)), and the user (`MainWindow::loadProject`) would do something with it. This API doesn't exist yet, which is why there are a bunch of `TODO` comments.

For now, dropping the error message into `qWarning()` is fine, but I'd encourage keeping the `TODO set error` comment here also. `Project` itself should *not* show a dialog; that should be up to the user of `Project` how it wants to handle errors. (The `qWarning()` should also go away once `Project` has a retrievable error message.)

**Comment by laurennlam** [comment]
Hi all,

I forgot to add an include to use the qWarning().
Can you wait until I correct my latest commit please ?

Laurenn LAM

**Comment by Purg** [comment]
Theoretically, the continuous integration build using Travis-CI should hit that compilation issue and fail, not allowing us to merge anyway.

**Comment by mleotta** [comment]
We will wait.  Travis-CI does not prevent a merge, I just lets you know when maybe it's not a good idea to merge because something is failing.

**Comment by Purg** [comment]
Ah, I wasn't sure how it was configured for this project. :)

**Comment by laurennlam** [comment]
The commit has been changed. Looks ok on my side.

Laurenn LAM

**Comment by mwoehlke-kitware** [comment]
@laurennlam, did you see my comment re: `show dialog?` vs. `set error` for the `TODO`? (Otherwise LGTM...)

File: gui/Project.cxx [hunk 1]
```diff
@@ -98,6 +99,11 @@ bool Project::read(QString const& path)
 
     return true;
   }
+  catch (kwiver::vital::config_block_exception e)
+  {
+    qWarning() << e.what(); // TODO dialog?
+    return false;
```
Comment (bastienjacquet) [inline gui/Project.cxx]: we should keep the `// TODO set error` for us to remember to handle the error when the API exists.

**Comment by laurennlam** [comment]
@mwoehlke-kitware , I pushed the modification :)

**Pull request merged**